### ISSUE TITLE
[Bug] TypeError: Multiple inheritance with NamedTuple is not supported

### ISSUE THREAD

**Issue by vitamins** [cgarciae/pypeln#71]
**Describe the bug**
Exception during import pypeln .
For testing this bug i am creating a new virtual environment and installing pypeln in it using the tool python-poetry.

**Minimal code to reproduce**
```
% poetry new test_pypeln
Created package test_pypeln in test_pypeln

% cd test_pypeln 

% poetry add pypeln
Creating virtualenv test-pypeln-vKtIMPjL-py3.9 in /home/lks/.cache/pypoetry/virtualenvs
Using version ^0.4.7 for pypeln

Updating dependencies
Resolving dependencies... (0.8s)

Writing lock file

Package operations: 12 installs, 0 updates, 0 removals

  • Installing pyparsing (2.4.7)
  • Installing attrs (20.3.0)
  • Installing more-itertools (8.7.0)
  • Installing packaging (20.9)
  • Installing pluggy (0.13.1)
  • Installing py (1.10.0)
  • Installing wcwidth (0.2.5)
  • Installing coverage (5.5)
  • Installing pytest (5.4.3)
  • Installing pytest-cov (2.11.1)
  • Installing stopit (1.1.2)
  • Installing pypeln (0.4.7)

% poetry run python
Python 3.9.2 (default, Feb 20 2021, 18:40:11) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pypeln
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/__init__.py", line 4, in <module>
    from . import thread
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/thread/__init__.py", line 34, in <module>
    from .api.concat import concat
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/thread/api/concat.py", line 8, in <module>
    from .to_stage import to_stage
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/thread/api/to_stage.py", line 5, in <module>
    from ..stage import Stage
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/thread/stage.py", line 8, in <module>
    from .queue import IterableQueue, OutputQueues
  File "/home/lks/.cache/pypoetry/virtualenvs/test-pypeln-vKtIMPjL-py3.9/lib/python3.9/site-packages/pypeln/thread/queue.py", line 15, in <module>
    class PipelineException(tp.NamedTuple, BaseException):
  File "/usr/lib/python3.9/typing.py", line 1881, in _namedtuple_mro_entries
    raise TypeError("Multiple inheritance with NamedTuple is not supported")
TypeError: Multiple inheritance with NamedTuple is not supported
>>> 

```

**Expected behavior**
import successful, no output

**Library Info**
pypeln (0.4.7)

**Comment by Tumornator** [comment]
This is due to a new implementation in Python 3.9
Temporary fix : Downgrade to Python 3.8/3,7

Here is a bug tracker on python website : https://bugs.python.org/issue43923

**Comment by cgarciae** [comment]
I believe this is closed with #78.
